src/main/java/com/github/black0nion/blackonionbot/bot/BotInformation.java

```diff
@@ -20,23 +20,29 @@
 import static com.github.black0nion.blackonionbot.misc.OperatingSystem.*;
 
 public class BotInformation {
+	private BotInformation() {}
 
-	private static final String datePattern = "dd.MM.yyyy HH:mm";
-	public static final SimpleDateFormat DATE_PATTERN = new SimpleDateFormat(datePattern);
-	public static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern(datePattern);
+	private static final String PATTERN = "dd.MM.yyyy HH:mm";
 
```

Comment:
I changed to osBean as sonarlight recommended and it is best practice.

Reply:
osBean is a constant and should thus be called `OS_BEAN`, according to the java conventions